### PR TITLE
Mercado Libre as known instance of the Trusted Committer pattern

### PR DESCRIPTION
**Pull request by privero-meli**
Add Mercado Libre as known instance of the Trusted Committer pattern

### CODE REVIEW

File: patterns/2-structured/trusted-committer.md [hunk 1]
```diff
@@ -127,6 +127,10 @@ This has been tried and proven successful at:
 
 - Nike
 - PayPal
+- Mercado Libre - We assign a section in the CONTRIBUTING file to inform who the TCs are.
+
+![image](https://user-images.githubusercontent.com/58035419/205659124-9fc7796d-3ee5-49b9-8c8c-1c7d24fa0488.png)
+
```
Comment (spier) [inline patterns/2-structured/trusted-committer.md]:
```suggestion
- Mercado Libre - adds a section in the `CONTRIBUTING.md` file to inform who the Trusted Committers are.

![image](https://user-images.githubusercontent.com/58035419/205659124-9fc7796d-3ee5-49b9-8c8c-1c7d24fa0488.png)
```

We try to stay away from using "we" in the Known Instances, as each of them is referring to a different company, which makes the use of "we" potentially confusing.

Also did some other rewording, please commit if you are happy with this.

**Review by spier — COMMENTED** [review]
Left one comment inline.

Also would you mind adding the image to folder `/assets/img`.
Then we can add a relative link to that from the markdown file.

File: patterns/2-structured/trusted-committer.md [hunk 2]
```diff
@@ -127,6 +127,9 @@ This has been tried and proven successful at:
 
 - Nike
 - PayPal
+- Mercado Libre - adds a section in the `CONTRIBUTING.md` file to inform who the Trusted Committers are.
+
+![Trusted Committer section](../../assets/img/mercadolibre-trusted-committers.png "Trusted Committer section")
```
Comment (spier) [inline patterns/2-structured/trusted-committer.md]:
```suggestion
![Trusted Committer section in CONTRIBUTING.md of Mercado Libre](../../assets/img/mercadolibre-trusted-committers.png "Trusted Committer section")
```

**Review by spier — COMMENTED** [review]
@privero-meli just noticed one more thing:
If I understand it correctly, the image is meant to be a mock of the `CONTRIBUTING.md` section in use at mercadolibre. Right?

However the text uses the wording "working group", which may be confusing in this context?

Also noticed that the text is pretty close to what we use in [the our own TC file](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/252a506fa03ee8020d466f8e0196f4ec44a38cb4/TRUSTED-COMMITTERS.md#trusted-committers) in this repo.

So maybe something that went unnoticed when copying the text?

**Comment by spier** [comment]
Btw not sure what the best wording in the context of an InnerSource repo is.

Here is [another example of a TC file](https://github.com/meltwater/gen_rmq/blob/master/TRUSTED-COMMITTERS.md) that I contributed to. However that one is for an open source project, so again the wording might not be 100% fitting for InnerSource.

**Comment by privero-meli** [comment]
> @privero-meli just noticed one more thing: If I understand it correctly, the image is meant to be a mock of the `CONTRIBUTING.md` section in use at mercadolibre. Right?
> 
> However the text uses the wording "working group", which may be confusing in this context?
> 
> Also noticed that the text is pretty close to what we use in [the our own TC file](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/252a506fa03ee8020d466f8e0196f4ec44a38cb4/TRUSTED-COMMITTERS.md#trusted-committers) in this repo.
> 
> So maybe something that went unnoticed when copying the text?

In Mercado Libre, to add a repository to Inner Source, the repository must have a CONTRIBUTOR file that includes this section.

Each repository has its owners and a reduced group of those owners are Trusted Committers. This is why we think "working group" makes sense.

And yes, it is clear that the section was taken from your CONTRIBUTING file 😀

File: patterns/2-structured/trusted-committer.md [hunk 3]
```diff
@@ -127,6 +127,9 @@ This has been tried and proven successful at:
 
 - Nike
 - PayPal
+- Mercado Libre - adds a section in the `CONTRIBUTING.md` file to inform who the Trusted Committers are.
+
+![Trusted Committer section in CONTRIBUTING.md of Mercado Libre](../../assets/img/mercadolibre-trusted-committers.png "Trusted Committer section")
```
Comment (spier) [inline patterns/2-structured/trusted-committer.md]:
```suggestion
![Trusted Committer section in CONTRIBUTING.md of Mercado Libre](../../assets/img/mercadolibre-trusted-committers.png "Trusted Committer section in CONTRIBUTING.md of Mercado Libre")
```

**Review by spier — APPROVED** [review]
Looks awesome!

We could consider to turn the image from Mercado Libre into text (a quote maybe?).
That way others can just copy and paste it, rather than having to re-type it all.
However we can leave that for a future PR, if we like that idea.

Thank you for this great contribution @privero-meli!

**Pull request merged**